staging/src/github.com/kubeedge/beehive/pkg/core/context/context_channel.go

```diff
@@ -262,11 +262,11 @@ func (ctx *ChannelContext) SendToGroupSync(moduleType string, message model.Mess
 		case <-sendTimer.C:
 			cleanup()
 			if timeoutCounter != 0 {
-				errInfo := fmt.Sprintf("timeout to send message, several %d timeout when send", timeoutCounter)
+				errInfo := fmt.Sprintf("Timeout to send message, several %d timeout when send", timeoutCounter)
 				return fmt.Errorf(errInfo)
```

Comment:
```suggestion
				return fmt.Errorf(Timeout to send message, several %d timeout when send", timeoutCounter)
```
It is recommended to combine the two methods